data/ralphsulit.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Ralph Sulit",
+  "bio": "Frontend Developer who loves React",
+  "avatar": "https://github.com/ralphsulit.png",
+  "links": [
+    {
+      "name": "Facebook",
+      "url": "https://www.facebook.com/ralphlsulit",
+      "icon": "FaFacebook"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/ralph-sulit/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/ralphsulit",
+      "icon": "FaGithub"
+    },
```

```suggestion
    }
```
This is a trailing comma which isnot required.  Please remove it.